src/main/java/org/folio/linked/data/mapper/dto/monograph/instance/sub/CarrierMapperUnit.java

```diff
@@ -16,14 +16,24 @@
 @MapperUnit(type = CATEGORY, predicate = CARRIER, dtoClass = Category.class)
 public class CarrierMapperUnit extends CategoryMapperUnit {
 
+  private static final String CARRIER_TYPE_LINK_PREFIX = "http://id.loc.gov/vocabulary/carriers/";
+
   public CarrierMapperUnit(CoreMapper coreMapper, HashService hashService) {
-    super(coreMapper, hashService, (category, destination) -> {
-      if (destination instanceof Instance instance) {
-        instance.addCarrierItem(category);
-      }
-      if (destination instanceof InstanceReference instance) {
-        instance.addCarrierItem(category);
-      }
-    }, CATEGORY);
+    super(coreMapper, hashService, CATEGORY);
+  }
+
+  @Override
+  protected void addToParent(Category category, Object parentDto) {
+    if (parentDto instanceof Instance instance) {
+      instance.addCarrierItem(category);
+    }
+    if (parentDto instanceof InstanceReference instance) {
```

Comment:
maybe better to name variable as ```instanceReference``` (same for ```workReference```)
I've noticed that same variable name causes some confusion